code/modules/research/designs/mecha_designs.dm

```diff
@@ -253,8 +218,8 @@
 	id = "mech_rcd"
 	build_type = MECHFAB
 	build_path = /obj/item/mecha_parts/mecha_equipment/rcd
-	materials = list(MAT_METAL=30000,MAT_GOLD=20000,MAT_PLASMA=25000,MAT_SILVER=20000)
-	construction_time = 1200
+	materials = list(MAT_METAL = 30000, MAT_GLASS = 20000, MAT_GOLD = 20000, MAT_SILVER = 20000, MAT_PLASMA = 25000)
+	construction_time = 200
```

Comment:
rcd construction time change, 1200 is obv a typo no one sane would think this is right

Reply:
same for this